Packs/EDL/Integrations/EDL/README.md

```diff
@@ -1,13 +1,22 @@
 This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed). The feed content can then be sent to 3rd parties for improving their security alignment.
 
+## PAN-OS EDL Management to PAN-OS EDL Service migration steps
+This integration hosts the EDL on the Cortex XSOAR server. Please follow these steps to migrate your EDLs.
```

Comment:
I would replace the beginning of the sentence with `Unlike EDL Management, EDL Service hosts the EDL on...`
This way we explain one of the main differences (hosting the list locally and not with a remote server).